sdk/tbrv3/solana/package.json

```diff
@@ -0,0 +1,26 @@
+{
+  "name": "solana-token-bridge-relayer-sdk",
```

Comment:
I think this should be namespaced to "@xlabs-xyz" so that we can push it to npm for connect. So on naming, should we go with `@xlabs-xyz/solana-arbitrary-token-transfers` ?

Reply:
Yeah, we should do the same for the EVM package.

Reply:
I will do that in the next commit, with the full SDK, can we merge already?

Reply:
Done